core/core-db-entities/src/main/java/com/sflpro/notifier/db/entities/notification/email/EmailNotification.java

```diff
@@ -34,6 +35,10 @@
     @Column(name = "locale")
     private Locale locale;
 
+    @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
why you are using cascade property?

Reply:
I know, we are performing no other operations than capturing the data, I just put it there thinking: why not? I'll remove it.